### PR TITLE
[Docs] - Add Warning about Version Downgrade and Alerting to Grafana …

### PR DESCRIPTION
**Pull request by SeamusGrafana**
…v9.4 Upgrade Instructions

Add Warning about Version Downgrade and Alerting to Grafana v9.4 Upgrade Instructions

**What is this feature?**

This updated the v9.4 Grafana Upgrade instructions to add a warning about a Database change.

**Why do we need this feature?**

To let users know about a new change in the Database that is not backwards compatible, and may cause issues should they downgrade for any reason.

**Who is this feature for?**

Readers of our Documentation, anyone that upgrades to v9.4 of Grafana

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/upgrade-guide/upgrade-v9.4/index.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ weight: 1995
 ## Technical notes
 
 There are no additional upgrade steps to complete for this version of Grafana.
+
+> Note: The upgrade to Grafana v9.4 makes changes to the Grafana Database for Grafana Alerting that are backwards incompatible, as such once upgraded to v9.4 of Grafana, your Grafana instance should not be downgraded to an earlier version. Doing so could cause issues with managing your Grafana Alerts.
```
Comment (chri2547) [inline docs/sources/upgrade-guide/upgrade-v9.4/index.md:21]:
```suggestion
The upgrade to Grafana v9.4 includes changes to the Grafana database for Grafana alerting that are not backward compatible. As a result, when you upgrade to Grafana v9.4, do not downgrade your Grafana instance to an earlier version. Doing so might cause issues with managing your Grafana alerts.
```

File: docs/sources/upgrade-guide/upgrade-v9.4/index.md [hunk 2]
```diff
@@ -17,3 +17,5 @@ weight: 1995
 ## Technical notes
 
 There are no additional upgrade steps to complete for this version of Grafana.
```
Comment (chri2547) [inline docs/sources/upgrade-guide/upgrade-v9.4/index.md:19]:
```suggestion
```

**Review by chri2547 — CHANGES_REQUESTED** [review]
Added suggestions. This is a really important piece of information and perfect for the technical notes section of the upgrade guide! Thank you very much for your contribution. After you've made the changes, please ping me for re-review.

**Review by chri2547 — APPROVED** [review]
LGTM, thank you!

**Comment by SeamusGrafana** [comment]
> Added suggestions. This is a really important piece of information and perfect for the technical notes section of the upgrade guide! Thank you very much for your contribution. After you've made the changes, please ping me for re-review.

Hi @chri2547 

Thanks for the review, changes accepted and committed.

**Comment by chri2547** [comment]
> > Added suggestions. This is a really important piece of information and perfect for the technical notes section of the upgrade guide! Thank you very much for your contribution. After you've made the changes, please ping me for re-review.
> 
> Hi @chri2547
> 
> Thanks for the review, changes accepted and committed.

@SeamusGrafana , looks like you need to run prettier. Do you know how to do that? Then we can mergel

**Pull request merged**